### PR TITLE
zh-tw: minor fixes to the content category article

### PR DESCRIPTION
**Pull request by yin1999**
### Description

minor fixes to the content category article

### Related issues and pull requests

prepare for: #18007

### CODE REVIEW

File: files/zh-tw/web/html/content_categories/index.md [hunk 1]
```diff
@@ -9,87 +9,235 @@ slug: Web/HTML/Content_categories
...
 
 ![Content_categories_venn.png](content_categories_venn.png)
 
 ## 主要內容類型
 
-### 資訊元內容（Metadata content）
+### 資訊元內容
 
-屬於*元資訊內容*類型的元素修飾該文件其餘部分的陳述或行為、與其他文件建立連結、或是傳達其他*外來*（out of band）訊息。
+屬於*元資訊內容*（Metadata content）類型的元素修飾該文件其餘部分的陳述或行為、與其他文件建立連結、或是傳達其他*外來*（out of band）訊息。
```
Comment (jasonren0403) [inline files/zh-tw/web/html/content_categories/index.md:22]: “元資訊內容” or “資訊元內容” ?

File: files/zh-tw/web/html/content_categories/index.md [hunk 2]
```diff
@@ -9,87 +9,235 @@ slug: Web/HTML/Content_categories
...
+- {{HTMLElement("sub")}}
+- {{HTMLElement("sup")}}
+- {{SVGElement("svg")}}
+- {{HTMLElement("template")}}
+- {{HTMLElement("textarea")}}
+- {{HTMLElement("time")}}
+- {{HTMLElement("u")}}
+- {{HTMLElement("var")}}
+- {{HTMLElement("video")}}
+- {{HTMLElement("wbr")}} -空白字符在內的純文本
```
Comment (jasonren0403) [inline files/zh-tw/web/html/content_categories/index.md:186]:
```suggestion
- {{HTMLElement("wbr")}} 
- 包含空白字符在內的純文本
```

File: files/zh-tw/web/html/content_categories/index.md [hunk 3]
```diff
@@ -9,87 +9,235 @@ slug: Web/HTML/Content_categories
...
+- {{HTMLElement("img")}}，如果 [`usemap`](/zh-TW/docs/Web/HTML/Element/img#usemap) 屬性存在。
+- {{HTMLElement("input")}}，如果 [`type`](/zh-TW/docs/Web/HTML/Element/input#type) 屬性不是隱藏狀態。
+- {{HTMLElement("object")}}，如果 [`usemap`](/zh-TW/docs/Web/HTML/Element/object#usemap) 屬性存在。
+- {{HTMLElement("video")}}，如果 [`controls`](/zh-TW/docs/Web/HTML/Element/video#controls) 屬性存在。
 
-### 捫及內容（Palpable content）
+### 捫及內容
 
-不是空白或隱藏的內容稱為捫及。屬於流內容或是 Phrasing content 模型的元素最少要有一個捫及的節點。
+不是空白或隱藏的內容稱為捫及（Palpable content）。屬於流內容或是段落型內容模型的元素最少要有一個捫及的節點。
```
Comment (jasonren0403) [inline files/zh-tw/web/html/content_categories/index.md:236]:
```suggestion
不是空白或隱藏的內容稱為捫及內容（Palpable content）。屬於流內容或是段落型內容模型的元素最少要有一個捫及的節點。
```

File: files/zh-tw/web/html/content_categories/index.md [hunk 4]
```diff
@@ -105,30 +253,29 @@ slug: Web/HTML/Content_categories
...
-<p>我們認為下面這些真理是<del><em>神聖不可否認</em></del><ins>不言而喻的。</ins></p>
+```html
+<p>
+  我們認為下面這些真理是<del><em>神聖不可否認</em></del
+  ><ins>不言而喻的。</ins>
+</p>
 ```
 
-這果這些元素被刪掉的話，這個分段依然在 HTML 有效（if not correct English）
+這果這些元素被刪掉的話，這個分段依然在 HTML 有效（如果不修正語法）
```
Comment (jasonren0403) [inline files/zh-tw/web/html/content_categories/index.md:277]:
```suggestion
如果這些元素被刪掉的話，這個分段依然在 HTML 有效（如果不修正語法）。
```

**Review by jasonren0403 — COMMENTED** [review]
一些建议

File: files/zh-tw/web/html/content_categories/index.md [hunk 5]
```diff
@@ -9,87 +9,235 @@ slug: Web/HTML/Content_categories
...
+- 主要內容類型（Main content category）描述了許多元素共享的常見內容規則（content rule）。
+- 表單相關內容類型（Form-related content category）描述了表單相關元素的內容規則。
+- 特別內容類型（Specific content category）描述了只有少數元素共享的內容規則，有時甚至只有特定上下文。
 
 ![Content_categories_venn.png](content_categories_venn.png)
 
 ## 主要內容類型
 
-### 資訊元內容（Metadata content）
+### 資訊元內容
```
Comment (yin1999) [inline files/zh-tw/web/html/content_categories/index.md:20]:
```suggestion
### 元資訊內容
```

**Review by jasonren0403 — APPROVED** [review]
LGTM

**Pull request merged**